pkg/reconciler/shared/tektonconfig/upgrade/post_upgrade.go

```diff
@@ -34,6 +35,10 @@ import (
 	"k8s.io/client-go/rest"
 )
 
+const (
+	keep = 2
```

Comment:
nit: `keep` variable name looks too generic, please rename it with meaningful, something like: `taskVersiondRetentionCount`